src/unordered-list/index.css

```diff
@@ -0,0 +1,22 @@
+/**
+ * @license EUPL-1.2
+ * Copyright (c) 2021 Robbert Broersma
+ * Copyright (c) 2021 Gemeente Utrecht
+ */
+
+.utrecht-unordered-list {
+  font-family: var(--utrecht-document-font-family, inherit);
+  margin-block-end: 1em;
+}
+
+.utrecht-unordered-list--nested {
+  margin-inline-start: 1.5em;
+}
+
+.utrecht-unordered-list__item {
```

Comment:
Should these values be more configurable? For example for `bullet-rood.svg`? Or are these going to be changed into configurations once there is another value

Reply:
I have refactored the implementation to use `::marker` and now the color of the bullet is configurable with the `--utrecht-unordered-list-marker-color` custom property.